concent_api/core/message_handlers.py

```diff
@@ -653,13 +657,14 @@ def get_clients_eth_accounts(task_to_compute: message.tasks.TaskToCompute):
     return (requestor_eth_address, provider_eth_address)
 
 
-def handle_send_force_payment(client_message: message.concents.ForcePayment) -> message.concents.ForcePaymentCommitted:  # pylint: disable=inconsistent-return-statements
+def handle_send_force_payment(client_message: message.concents.ForcePayment) \
```

Comment:
We don't brake signatures this way, please check coding-style.

Reply:
done